src/ir.h

```diff
@@ -130,6 +128,22 @@ struct Block {
   Expr* first;
 };
 
+struct Catch {
+  WABT_DISALLOW_COPY_AND_ASSIGN(Catch);
+  Catch() = delete;
```

Comment:
Doesn't hurt, but isn't this implicitly deleted because of the constructors below?